### DISCUSSION TITLE
GitHub pages CNAME file

### DISCUSSION THREAD

**Discussion by tommerty** [jpanther/congo#426] [Q&A] [answered]
Hi there,
Just want to say thanks for this project, it's making me want to work with Hugo!

Here's my problem: I have a custom domain set as blog.tommerty.click and followed your GitHub instructions [here](https://jpanther.github.io/congo/docs/hosting-deployment/#github-pages)
The problem I'm having however is that the `gh-pages` branch always loses it's `CNAME` file for my custom domain. I've tried adding it on both main and this branch, but it always gets deleted after a push and have to remember to manually go in and fix it, otherwise I get the GH  `404 - There isn't a GitHub Pages site here` error.

I can't find anything outlining how to ensure the custom domain is set, any advise would be great.
You can find my repo at https://github.com/tommerty/mywebsite

Thanks in advance!

**Comment by tommerty** [comment]
Bumping to hope to get a response

**Comment by code-not-found** [comment, marked as answer]
Was running into the same issue. Adding the CNAME file to the static folder fixed it for me: [Custom domain gets deleted on deploy](https://github.com/tschaub/gh-pages/issues/236)

**Reply by tommerty** [reply]
Thanks for that! So when you say to put it in the static folder, I don't have one already so I'm unsure where to add it. I followed the docs and have a `public` folder in `main` should I put a `static` folder in the `gh-pages` and place the file in there? My repo is [here](https://github.com/tommerty/mywebsite) if that helps.
Thanks!

**Comment by tommerty** [comment]
Ah, managed to sort it! Thanks
